docs/GettingStarted.md

```diff
@@ -246,7 +246,7 @@ As you can see from when you check the results file the licenses detected by `Sc
 the packages. This is because we scanned a small and well-maintained package in this example, but if you run the scan on
 a bigger project you will see that `ScanCode` often finds more licenses than are declared by the packages.
 
-The `scanner` appends its result to the analyzer result file and writes it to the file
+The `scanner` puts its results next to the analyzer result in the ORT result file and writes a new ORT result file to
```

Comment:
Sounds a bit like it would modify the existing file and write a new one. Maybe:

> The `scanner` writes a new ORT result file to ... which contains the analyzer result and the scan results. Keeping the analyzer result and scan results in the same file ensures traceability for what input was used to create the scan results. If the input file already contains scan results they are overwritten by the new scan results.

Reply:
Comments addressed using slightly different wording.